test/unit/get-new-logger.test.js

```diff
@@ -0,0 +1,141 @@
+/**
+ * (C) Copyright IBM Corp. 2019, 2021.
```

Comment:
```suggestion
 * (C) Copyright IBM Corp. 2019, 2022.
```
I will serve as @pyrooka for this PR :joy:

Reply:
Making fun of me, huh? 😂

Reply:
Making fun of me, huh? 😂